package.json

```
@@ -27,19 +27,30 @@
     "typecheck:shared": "tsc --noEmit --project packages/shared",
     "typecheck:frontend": "tsc --noEmit --project packages/frontend",
     "typecheck:backend": "cd packages/backend && npm run typecheck",
-    "typecheck": "npm run typecheck:shared && npm run typecheck:frontend && npm run typecheck:backend"
+    "typecheck": "npm run typecheck:shared && npm run typecheck:frontend && npm run typecheck:backend",
+    "prepare": "husky"
   },
   "resolutions": {
     "string-width": "^4.2.0",
     "wrap-ansi": "^7.0.0",
     "postman-code-generators": "1.10.1"
   },
+  "lint-staged": {
+    "*.{js,ts,tsx,svelte}": [
+      "node --max-old-space-size=6144 node_modules/eslint/bin/eslint.js --cache --fix",
```

suggestion: we don't need more memory on ai-lab compared to podman-desktop to run the linter
```suggestion
      "eslint --cache --fix",
```